.gitignore

```diff
@@ -74,3 +74,5 @@ lib*.pc
 /tmp_install/
 /.cache/
 /install/
+# macOS hidden files
```

Comment:
Remove this line and move the `.DS_Store` to the line 62(which below the `# Global excludes across all subdirectories`)? As you can see the `*.dylib` is the macOS only, but it was not move alone.

Reply:
Good catch, thanks!

Reply:
PTAL, thanks.